src/flat_file.cpp

```diff
@@ -229,19 +227,27 @@ Status::type read_flat_file_trackcpp(std::istream& fp, Accelerator& accelerator)
     if (cmd.compare("angle_out")   == 0) { ss >> e.angle_out; continue; }
     if (cmd.compare("rescale_kicks")   == 0) { ss >> e.rescale_kicks; continue; }
     if (cmd.compare("t_in")      == 0) { for(auto i=0; i<6; ++i) ss >> e.t_in[i];  continue; }
+    e.reflag_t_in();
```

Comment:
@VitorSouzaLNLS why is this re-flag out of the if statement?

Reply:
Since reflag_r_in and reflag_r_out demand staying out of the "if"s, I also wrote the reflag_t_in and reflag_t_out outside for standardization. Would it be better to keep them inside?